website/content/gateway/api_reference/transaction_status/transaction_status_codes_table.md

```diff
@@ -4,27 +4,27 @@ date: 2022-04-13T12:37:22+02:00
-|          | 40120 | Invalid CSC                        | ✓    |         |        |      |        |
-|          | 40130 | Invalid expire date                | ✓    |         |        |      | ✓      |
-|          | 40135 | Card expired                       | ✓    |         |        |      | ✓      |
-|          | 40140 | Invalid currency                   | ✓    |         |        |      | ✓      |
-|          | 40150 | Invalid text on statement          | ✓    | ✓       | ✓      |      | ✓      |
-|          | 40200 | Clearhaus rule violation           | ✓    | ✓       | ✓      | ✓    | ✓      |
-|          | 40300 | 3-D Secure problem                 | ✓    |         |        |      |        |
-|          | 40310 | 3-D Secure authentication failure  | ✓    |         |        |      |        |
-|          | 40400 | Backend problem                    | ✓    |         |        | ✓    | ✓      |
-|          | 40410 | Declined by issuer or card scheme  | ✓    |         |        | ✓    | ✓      |
-|          | 40411 | Card restricted                    | ✓    |         |        |      | ✓      |
-|          | 40412 | Card lost or stolen                | ✓    |         |        |      | ✓      |
-|          | 40413 | Insufficient funds                 | ✓    |         |        |      | ✓      |
-|          | 40414 | Suspected fraud                    | ✓    |         |        |      | ✓      |
-|          | 40415 | Amount limit exceeded              | ✓    |         |        |      | ✓      |
-|          | 40416 | Additional authentication required | ✓    |         |        |      | ✓      |
-|          | 40420 | Merchant blocked by cardholder     | ✓    |         |        |      | ✓      |
-|          | 50000 | Clearhaus error                    | ✓    | ✓       | ✓      | ✓    | ✓      |
+| Status   | code  | Meaning                            | Auth | Capture | Refund | Void | Credit | Debit  | Debit Refund |
+|----------|-------|------------------------------------|------|---------|--------|------|--------|--------|--------------|
+| Approved | 20000 | Approved                           | ✓    | ✓       | ✓      | ✓    | ✓      | ✓      | ✓            |
+| Declined | 40000 | General input error                | ✓    | ✓       | ✓      | ✓    | ✓      | ✓      | ✓            |
+|          | 40110 | Invalid card number                | ✓    |         |        |      | ✓      | ✓      |              |
+|          | 40111 | Unsupported card scheme            | ✓    |         |        |      | ✓      | ✓      |              |
+|          | 40120 | Invalid CSC                        | ✓    |         |        |      |        | ✓      |              |
+|          | 40130 | Invalid expire date                | ✓    |         |        |      | ✓      | ✓      |              |
+|          | 40135 | Card expired                       | ✓    |         |        |      | ✓      | ✓      |              |
```

Comment:
Same story as for 40110 / invalid card number. A debit and debit refund pair might cross end of month, potentially entailing card expiry. We could receive ISO 8583 response code 54 from upstream meaning card expired. Not very realistic, and perhaps Visa even allows debit refunds on expired cards.

Reply:
```suggestion
|          | 40135 | Card expired                       | ✓    |         |        |      | ✓      | ✓      | ✓            |
```